cypress.config.js

```diff
@@ -10,10 +10,17 @@ module.exports = defineConfig({
           const email = faker.internet.email();
           const randomNumber = Math.floor(Math.random(1000) * 1000);
           return {
-            username: faker.name.firstName() + randomNumber,
+            username: faker.person.firstName() + randomNumber,
             email: email.toLowerCase(),
             password: '12345Qwert!'
```

Comment:
It's a good practice to avoid hardcoding passwords, even for test data. You might want to consider generating random passwords as well, to better simulate real-world scenarios.